src/lib/form-builder/components/GlobalVariableSelect.tsx

```
@@ -0,0 +1,151 @@
+                            ) : (
+                                <ul className="space-y-1" ref={listRef}>
+                                    {items.map((item, index) => (
+                                        <li
+                                            key={item.key}
+                                            className={cn(
+                                                "relative flex cursor-pointer select-none items-center rounded-sm px-2 py-1.5 text-sm outline-none transition-colors",
+                                                index === selectedIndex ? "bg-accent text-accent-foreground" : "hover:bg-accent/50"
+                                            )}
+                                            // Mouse click selection
+                                            onClick={() => onSelect(item)}
+                                        >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)